common/text/text_structure.cc

```diff
@@ -154,6 +154,14 @@ LineColumnRange TextStructureView::GetRangeForText(
   return {GetLineColAtOffset(from), GetLineColAtOffset(to)};
 }
 
+bool TextStructureView::ContainsText(absl::string_view text) const {
+  const auto from = std::distance(Contents().begin(), text.begin());
+  const auto to = std::distance(Contents().begin(), text.end());
+  if (from < 0) return false;
+  if (to > static_cast<int>(Contents().length())) return false;
```

Comment:
If you still need a static cast here (I suspect because of signed, unsigned), better used `int64_t`

Reply:
Fixed